src/main/java/com/denizenscript/clientizen/objects/properties/entity/EntitySneaking.java

```diff
@@ -0,0 +1,56 @@
+package com.denizenscript.clientizen.objects.properties.entity;
+
+import com.denizenscript.clientizen.access.KeyBindingMixinAccess;
+import com.denizenscript.clientizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import net.minecraft.client.MinecraftClient;
+
+public class EntitySneaking extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name sneaking
+    // @input ElementTag(Boolean)
+    // @description
+    // Whether an entity is sneaking.
+    // For most entities this just makes the name tag less visible, and doesn't actually update the pose.
+    // -->
+
+    public static boolean describes(EntityTag entity) {
+        return true;
+    }
+
+    @Override
+    public ElementTag getPropertyValue() {
+        return new ElementTag(getEntity().isSneaking());
+    }
+
+    @Override
+    public boolean isDefaultValue(ElementTag value) {
+        return !value.asBoolean();
+    }
+
+    @Override
+    public void setPropertyValue(ElementTag value, Mechanism mechanism) {
+        if (!mechanism.requireBoolean()) {
+            return;
+        }
+        boolean sneaking = value.asBoolean();
+        getEntity().setSneaking(sneaking);
+        if (getEntity() == MinecraftClient.getInstance().player) {
+            KeyBindingMixinAccess keyBindingMixinAccess = (KeyBindingMixinAccess) MinecraftClient.getInstance().options.sneakKey;
+            keyBindingMixinAccess.forceSetPressed(sneaking);
+            keyBindingMixinAccess.disableUntilPress();
```

Comment:
Considering that the only time you call these two methods is here, is there any reason they should be split in two? It might be better to just have one `forceSetPressed` method in `KeyBindingMixinAccess` that sets `clientizen$disabled` to false.

Reply:
I don't think that'd make sense; currently they're only used there but it's technically a generic thing that can be used anywhere - we'd have to name the method `forceSetPressedAndDisable` or something, as I'd expect `forceSetPressed` to just force set the pressed value.